docs/editing.md

```diff
@@ -437,3 +437,90 @@ Try to check in on your JOSS submissions twice per week, even if only for 5 minu
+
+All new editors at JOSS have an onboarding call with an Editor-in-Chief. You can use the structure below to make sure you highlight the most important aspects of being an editor.
+
+**Thing to check before the call**
+
+- Have they reviewed or published in JOSS before? If not, you'll need to spend significantly more time explaining how the review process works.
+- Check on their research background (e.g., what tracks they might edit most actively in).
+- Make sure to send them the [editorial guide](https://joss.readthedocs.io/en/latest/editing.html) to read before the call.
+
+### The onboarding call
+
+**Preamble/introductions**
+
+- Welcome! Thank them for their application to join the team.
+- Point out that this isn't an interview. Rather, this is an informational call designed to give the candidate the information they need to make an informed decision about editing at JOSS.
+- 90-day trial period/try out. Editor or JOSS editorial board can decide to part ways after that period.
+- No strict term limits. Some editors have been with us for 7+ years, others do 1-2 years. Most important thing is to be proactive with your editing responsibilities.
+- Confirm with them their level of familiarity with JOSS/our review process.
+- Point out that they *do not* need to make a decision on the call today. They are welcome to have a think about joining and get back to us.
+
+**Share your screen**
+
+- Visit JOSS (https://joss.theoj.org)
+- Pick a recently-published paper (you might want to identify this before the call one that shows off the review process well).
+- Show the paper on the JOSS site, and then go to the linked review issue.
+- Explain that there are *two* issues per submission – the pre-review issue and the main review issue.
+
+**The pre-review issue**
+
+- The 'meeting room for the paper'. Where author meets editor, and reviewers are identified.
+- Note that the EiC may have initiated a scope review. The editor should not start editing until this has completed. Also, editors are able to query the scope (as are reviewers) if they think the EiC should have (but didn't). 
+- Walk them through what is happening in the pre-review issue...
+- Editor is invited (likely with GitHub mention but also via email invite (`@editorialbot invite @editor as editor`))
+- Once editor accepts they start looking for reviewers.
+
+**Finding reviewers**
+
+- Explain that this is one of the more time-intensive aspects of editing.
+- Explain where you can look for editors (your own professional network, asking the authors for recommendations, the [reviewers application](https://reviewers.joss.theoj.org/), similar papers identified by Editorialbot, )
+- Point out that we have a minimum of two reviewers, but if more than that accept (e.g., 3/4 then take them all – this gives you redundancy if one drops out).
+- Don't invite one reviewer at a time! If you do this, it may take many weeks to find two reviewers who accept. Try 3/4/5 invites simultaneously.
```

Comment:
```suggestion
- Don't invite only one reviewer at a time! If you do this, it may take many weeks to find two reviewers who accept. Try 3/4/5 invites simultaneously.
```